pkgs/build-support/setup-hooks/set-source-date-epoch-to-latest.sh

```diff
@@ -1,5 +1,9 @@
 updateSourceDateEpoch() {
     local path="$1"
+    # Avoid passing option-looking directory to find. The example
+    # is diffoscope-269:
```

Comment:
Stupid line-wrapping suggestion:

```suggestion
    # Avoid passing option-looking directory to find. The example is diffoscope-269:
```